yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Add this file to the .gitignore

Reply:
Shouldn't lock files get committed to the repo?

Reply:
Yeah you're right. You can ignore this as well